README.md

```diff
@@ -1 +1,84 @@
-# Career Scraper
+
+# LinkedIn Profile Scraper
+
+The LinkedIn Profile Scraper is a tool designed to extract information from the profile page of a given URL on LinkedIn. It provides an automated way to gather data from LinkedIn profiles, such as personal details, work experience, education, skills, and more.
+
+## Features
+
+Automatically extracts the information like, name, location, experience, education, skills of the profile page of the given URL and returns the output in JSON format.
+
+## Dependencies
+- **Python3**: Ensure that you have Python 3 installed on your system. You can download and install Python 3 from the official Python website: https://www.python.org.
+- **pip**: pip is the package installer for Python. It is usually installed by default when you install Python. However, make sure you have pip installed and it is up to date. You can check the version of pip by running the following command:
+  ```
+  pip --version
+  ```
+- **Selenium**: You can install it using pip by running the following command
+  ```
+  pip install selenium
```

Comment:
```suggestion
  pip install -r requirements.txt
```

Add a requirements.txt file. Also, remove .txt from .gitignore?

Reply:
requirements file is already there. 
I have removed .txt from .gitignore.